### DISCUSSION TITLE
adding Github repo connection

### DISCUSSION THREAD

**Discussion by Umair841** [argoproj/argo-helm#2049] [Q&A]
I am using this repo to setup argocd. after instalation i need to add github repo connection on UI.
is there any way to add those github repos while installing helm of Argocd ?

**Comment by yu-croco** [comment]
Hi @Umair841 ,
I think you can add the info of repos (e.g. GitHub, GitLab) at [this block](https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/values.yaml#L465). 🙋 
Also, known hosts are [here](https://github.com/argoproj/argo-helm/blob/main/charts/argo-cd/values.yaml#L317), if you need to configure.
